README.md

```diff
@@ -1,18 +1,20 @@
-# Aragon React Boilerplate
+# Aragon Buidler Boilerplate
 
 > 🕵️ [Find more boilerplates using GitHub](https://github.com/search?q=topic:aragon-boilerplate) |
 > ✨ [Official boilerplates](https://github.com/search?q=topic:aragon-boilerplate+org:aragon)
 
-React boilerplate for Aragon applications.
+Buidler + React boilerplate for Aragon applications.
 
-This boilerplate includes a fully working example app, complete with a background worker and a front-end in React (with Aragon UI). Also comes with a DAO Template which will allow for using your app to interact with other Aragon apps like the Voting app. You can read more about DAO Template [here](https://hack.aragon.org/docs/templates-intro).
+This boilerplate includes a fully working example app, complete with a background worker and a front-end in React (with Aragon UI).
+
+_Note: This is an experimental boilerplate for developing Aragon applications. For a more stable boilerplate, please use aragon-react-boilerplate._
 
 ## Usage
 
 To setup use the command `create-aragon-app`:
 
 ```sh
-npx create-aragon-app <app-name> react
+npx create-aragon-app <app-name> buidler
 ```
 
 ## Structure
```

Comment:
Before this "structure" section, it might be nice to include a "next steps" section with:

- Add code quality tools, like JS and contract linting
- Develop your [AragonApp contract]()
- Develop your [frontend]()
- Publish! (link to guide)

Let's also move the "Running your app" section to be above the "next steps" section.